pkg/validation/internal/csv.go

```diff
@@ -50,20 +49,11 @@ func validateCSV(csv *v1alpha1.ClusterServiceVersion) errors.ManifestResult {
 	return result
 }
 
-func parseCSVNameFormat(name string) (string, semver.Version, error) {
+func parseCSVNameFormat(name string) error {
 	if violations := k8svalidation.IsDNS1123Subdomain(name); len(violations) != 0 {
-		return "", semver.Version{}, fmt.Errorf("%q is invalid:\n%s", name, violations)
+		return fmt.Errorf("%q is invalid:\n%s", name, violations)
 	}
-	splitName := strings.SplitN(name, ".", 2)
-	if len(splitName) != 2 {
-		return "", semver.Version{}, fmt.Errorf("%q must have format: {operator name}.(v)X.Y.Z", name)
-	}
-	verStr := strings.TrimLeft(splitName[1], "v")
-	nameVer, err := semver.Parse(verStr)
-	if err != nil {
-		return "", semver.Version{}, fmt.Errorf("%q contains an invalid semver %q", name, splitName[1])
-	}
-	return splitName[0], nameVer, errors.Error{}
+	return errors.Error{}
```

Comment:
Any reason we're not returning `nil` here?

Reply:
Using `nil` should be fine - was just trying to stay consistent with the rest of the code